test/ex_doc/language/erlang_test.exs

```diff
@@ -161,6 +161,59 @@ defmodule ExDoc.Language.ErlangTest do
     end
   end
 
+  describe "autolink_doc/2 for extras" do
+    test "function", c do
+      assert autolink_extra("`foo:foo/0`", c) ==
+               ~s|<a href="foo.html#foo/0"><code class="inline">foo:foo/0</code></a>|
+    end
+
+    test "OTP function", c do
+      assert autolink_extra("`lists:reverse/1`", c) ==
+               ~s|<a href="https://www.erlang.org/doc/man/lists.html#reverse-1"><code class="inline">lists:reverse/1</code></a>|
+    end
+
+    test "type", c do
+      assert autolink_extra("`t:bar:t()`", c) ==
+               ~s|<a href="bar.html#t:t/0"><code class="inline">bar:t()</code></a>|
+    end
+
+    test "OTP type", c do
+      assert autolink_extra("`t:array:array()`", c) ==
+               ~s|<a href="https://www.erlang.org/doc/man/array.html#type-array"><code class="inline">array:array()</code></a>|
+    end
+
+    test "module", c do
+      assert autolink_extra("`foo`", c) ==
+               ~s|<a href="foo.html"><code class="inline">foo</code></a>|
+    end
+
+    test "OTP module", c do
+      assert autolink_extra("`rpc`", c) ==
+               ~s|<a href="https://www.erlang.org/doc/man/rpc.html"><code class="inline">rpc</code></a>|
+    end
+
+    test "bad function", c do
+      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+               assert autolink_extra("`bad:bad/0`", c) ==
+                        ~s|<code class="inline">bad:bad/0</code>|
+             end) =~ ~s|references function "bad:bad/0" but it is undefined or private|
+    end
+
+    test "bad type", c do
+      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+               assert autolink_extra("`t:bad:bad/0`", c) ==
+                        ~s|<code class="inline">t:bad:bad/0</code>|
+             end) =~ ~s|references type "bad:bad()" but it is undefined or private|
+    end
+
+    test "bad module", c do
+      assert ExUnit.CaptureIO.capture_io(:stderr, fn ->
+               assert autolink_extra("`does_not_exist`", c) ==
+                        ~s|<code class="inline">does_not_exist</code>|
+             end) =~ ~s|references module "does_not_exist" but it is undefined|
+    end
```

Comment:
Is emitting warnings for unknown modules too verbose? Code like <code>\`true\`</code> would trigger a warning

Reply:
Yes. :(